docs/add-ins/delegate-access.md

```diff
@@ -0,0 +1,94 @@
+
+## Configure the manifest
+
+You should set the [SupportsSharedFolders](/office/dev/add-ins/reference/manifest/supportssharedfolders) element to `true` under the parent element `DesktopFormFactor` so that your add-in is available in delegate access scenarios.
+
+> [!IMPORTANT]
+> The `SupportsSharedFolders` element is only available in the [Outlook add-ins preview requirement set](/office/dev/add-ins/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview) against Exchange Online.
+>
+> Add-ins that use this element cannot be published to AppSource or deployed via centralized deployment.
+
+The following example shows the `SupportsSharedFolders` element set to `true` in a section of the manifest.
+
+```XML
+<DesktopFormFactor>
+  <FunctionFile resid="residDesktopFuncUrl" />
+  <SupportsSharedFolders>true</SupportsSharedFolders>
+  <ExtensionPoint xsi:type="PrimaryCommandSurface">
+    <!-- information about this extension point -->
+  </ExtensionPoint>
+</DesktopFormFactor>
+```
+
+## Use in messages and appointments for compose and read modes
+
+You can get the item's shared properties by calling the [item.getSharedPropertiesAsync](/office/dev/add-ins/reference/objectmodel/preview-requirement-set/office.context.mailbox.item#getsharedpropertiesasyncoptions-callback) method. This returns a [SharedProperties](/javascript/api/outlook/office.sharedproperties) object that currently provides the delegate's permissions, the owner's email address, and the REST URL of the owner's mailbox.
+
+The following example shows how to get an item's shared properties and check if the delegate has **Write** permission.
+
+```js
+Office.context.mailbox.item.getSharedPropertiesAsync(callback);
+
+function callback (asyncResult) {
+  var sharedProperties = asyncResult.value;
+
+  // Check if user is a delegate before proceeding.
+  if (sharedProperties != null) {
+    console.log(JSON.stringify(sharedProperties));
+
+    var delegatePermissions = sharedProperties.delegatePermissions;
+
+    // Determine if user can do the expected action.
+    // E.g., do they have Write permission?
+    if ((delegatePermissions & MailboxEnums.Write) != 0 ) {
+      // Expected action...
+    }
+  }
+}
+```
+
+## Limitations and restrictions
+
+At present, delegate access is only available in the [Outlook add-ins preview requirement set](/office/dev/add-ins/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview) against Exchange Online. Add-ins that use the `SupportsSharedFolders` element in their manifest cannot be published to AppSource or deployed via centralized deployment as the delegate access feature APIs should not be used in production environments yet.
```

Comment:
Again, I don't understand what this means "against Exchange Online."  Does it makes sense to say "At present, delegate access is only available to run against Exchange Online?"